core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -64,6 +64,12 @@ oppia.controller('SettingsTab', [
     var CREATOR_DASHBOARD_PAGE_URL = '/creator_dashboard';
     var EXPLORE_PAGE_PREFIX = '/explore/';
 
+    // This is needed to make select2 re-load the dom if user have
+    // changed/added new value while publishing exploration.
+    $scope.showSelect2Dropdown = function() {
```

Comment:
This doesn't actually show anything, it just performs a check. So maybe we should call it isSettingsTabActive() instead.